source/config/__init__.py

```diff
@@ -64,7 +64,13 @@ def isInstalledCopy():
 
 def getInstalledUserConfigPath():
 	try:
-		return os.path.join(shlobj.SHGetFolderPath(0, shlobj.CSIDL_APPDATA), "nvda")
+		k = _winreg.OpenKey(_winreg.HKEY_LOCAL_MACHINE, NVDA_REGKEY)
+		configInLocalAppData = bool(_winreg.QueryValueEx(k, u"configInLocalAppData")[0])

```

Comment:
You interpreted my comment right, I updated it accordingly. Sorry for the misunderstanding.